lib/sasl/src/sasl.app.src

```diff
@@ -42,6 +42,6 @@
     {applications, [kernel, stdlib]},
     {env, []},
     {mod, {sasl, []}},
-    {runtime_dependencies, ["tools-2.6.14","stdlib-3.4","kernel-6.0",
+    {runtime_dependencies, ["tools-2.6.14","stdlib-4.1.1","kernel-6.0",
```

Comment:
The `stdlib` dependency should only be bumped to `stdlib-4.0` if the bump is only due to using `lists:uniq()`